src/test/java/hudson/remoting/ForkEBCDICRunner.java

```diff
@@ -12,4 +12,9 @@
         r.add(0,"-Dfile.encoding=CP037");
         return r;
     }
+
+    @Override
+    public String getName() {
```

Comment:
Oh look I have [two failed tests](https://jenkins.ci.cloudbees.com/job/libraries/job/remoting/147/testReport/)... and they have the same name... how on earth am I supposed to determine which is the one at fault... is it the `-fork` from `ForkRunner` or is it the `-fork` from `ForkEBCDICRunner`...

with this change you will worry no more 

(And when I was able to reproduce that failure locally it would only be for one of the forks at a time... because I have more cores and the threads were usually woken in the correct order to let the tests pass)